scripts/setup

```diff
@@ -9,14 +9,19 @@ cd $(git rev-parse --show-toplevel)
 python -m venv test/tiledbsoma
 source test/tiledbsoma/bin/activate
 
-# install python api and dependencies
+# clean
+rm -rf build dist
+rm -f apis/python/src/tiledbsoma/libtiledb.so.*
```

Comment:
note these end in `.dylib` on MacOS

Reply:
Right, that was updated in the [29e230a](https://github.com/single-cell-data/TileDB-SOMA/pull/304/commits/29e230a7ca74aefe9d2b72e55e88fcb84efb8453). Maybe the review picked up an old version?